File: src/lesson017/java/Main.java
package lesson017.java;

public class Main {
    public static void main(String[] args) {
        StringBuilder sb1 = new StringBuilder("Hello");
        StringBuilder sb2 = new StringBuilder("Hello");
        System.out.println(EqualSBInner.checkEqual(sb1, sb2));
    }
}

class EqualSBInner{
    static boolean checkEqual(StringBuilder sb1, StringBuilder sb2){
        return sb1.toString().equals(sb2.toString());
    }
}

File: src/lesson017/kotlin/Main.kt
package lesson017.kotlin

fun main(args: Array<String>) {
    println(EqualSBInner.checkEqual(StringBuilder("Hello"), StringBuilder("Hello")))
}

class EqualSBInner{
    companion object{
        fun checkEqual(sb1: StringBuilder, sb2: StringBuilder) = sb1.toString() == sb2.toString()
    }
}